### ISSUE TITLE
[Guide] Finding Metadata

### ISSUE THREAD

**Issue by MegaApplePi** [ppy/osu-wiki#965]
🆘 I was going to port the guide over from <https://osu.ppy.sh/community/forums/topics/249288> but the guide only goes over a song as an example and contains limiting steps. I would expand it but I'm no expert in this, so I'll paste my draft here. For those who knows how to do this, please help fix/add to it by including the steps you take into finding metadata.

```md
# Finding Metadata

Metadata in osu! is a set of data that describes the song of a [beatmap](/wiki/beatmap).
osu! uses this metadata in many places (e.g. song selection and searching on both osu!client and osu!web).

You put metadata in the `General` tab of the [song setup](/wiki/song_setup).

![](img/FM_1.jpg "Song Setup: General tab")

## Getting Basic Information

First, you need to know some basic information of the song. Of course, you may have found it somewhere like YouTube or SoundCloud and had obtained the audio by any means. More than likely, you already know what song you want to map or you've stumbled upon a song when searching. **In such case where you've stumbled upon a song, look around the site for some basic information on the song.**

**Nobody** will be able to figure out the song title exactly while listening to it for the first few times. However, if you are lucky and do guess what the song is it won't hurt to look up the song information to make sure.

![](img/FM_2.jpg)

Take that information and above and put it into google! **But**, if you happen to find the song information in the video. **You** can use it! (but make sure you check the official site to see if it uses (TV Size) etc.

## Searching

Now that you've searched in Google, a list of results should pop up! You can use Reference sites to help you find the official website. From there try to find the Japanese Romanticized version of the name of the anime. PUT that inside the search box (If you are Japanese and already know it, just type it in). Once there you should be able to find the official website! (you can add official website to your search) to narrow it down.

The video has the artist: sweet ARMS, use that for the artist.

In this case, the Japanese name for Date a Live II is デート・ア・ライブⅡ , put that in google, the first result that shows up should be it, which is <http://date-a-live-anime.com/>.

Look around and you should find it

<http://date-a-live-anime.com/book/cd03.html>

![](img/FM_3.jpg)

The title is : 01. Trust in you (TV Size)

You can use ANY source, as long as its Official.

## tl;dr

You can use **any** source, as long as that source an **official source**.

## Trivia

- Original forum post: [\[Guide\] Metadata](/community/forums/topics/249288) by [Tari](/users/1779172)
```

**Comment by sr229** [comment]
Auto-assigning to self
